src/main/osd/osd_elements.c

```diff
@@ -1279,25 +1283,23 @@ static void osdElementWarnings(osdElementParms_t *element)
         {
             tfp_sprintf(element->buff, "LAUNCH");
         }
-
-        // Blink the message if the throttle is within 10% of the launch setting
-        if ( calculateThrottlePercent() >= MAX(currentPidProfile->launchControlThrottlePercent - 10, 0)) {
-            SET_BLINK(OSD_WARNINGS);
-        }
```

Comment:
This looks like a rebase problem as it's removing a recently added feature.

Reply:
Doh...